requirements.txt

```diff
@@ -10,3 +10,4 @@ python-manilaclient
 python-neutronclient
 python-novaclient
 python-zunclient
+ipywidgets
```

Comment:
Is this a leftover from the widgets PR? Doesn't look like it's used in this PR.

Reply:
Yeah this is a mistake, let me remove it!